build/scripts/DownloadDotNetRuntimeInstaller.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
```

Comment:
Is this an open ended string or a limited allowed list? Seems like the latter, in which case shouldn't it be an enum (well, Powershell's expression of an enum)?

Reply:
Aggregator takes a string separated by commas. "x64,x86"

Reply:
But for this installer, it is only invoked for one platform at a time.